tests-behat/useraction.feature

```diff
@@ -1,14 +1,14 @@
-@user-action-tag
+@exec-user-action
```

Comment:
we really need a CI test for tags if they are 1:1, otherwise we might confuse users why one name used one way work and not in another